Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -325,11 +325,20 @@ define([
                 if (value) {
                     that._eventHandler.setInputAction(function(e) {
                         var picked = scene.pick(e.endPosition);
-                        if (picked instanceof Cesium3DTileFeature) {
+                        if (picked && defined(picked.content)) {
                             that.feature = picked;
+                            var pos;
+                            e.endPosition.x += 10;
+                            e.endPosition.y -= 10;
```

Comment:
Instead of offsetting the pick position, you could add a `pixelOffset` to the label. This may fix some of the other issues, but we'll see.